.github/workflows/build_docker_container_and_push_to_ghcrio.yml

```diff
@@ -0,0 +1,69 @@
+name: deploy
+
+on:
+  push:
+    branches:
+      - main
+      - v2
+jobs:
+  tag_and_dockerise:
+    # When merging to one of the branches above and the commit message matches
+    if: "startsWith(github.event.head_commit.message, 'Bump version:')"
+    name: Tag and dockerise
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
```

Comment:
How does this invoke the bump script?

Reply:
That's just a default, the `./bump` script is invoked locally, that bumps the version in `package.json` and `package-lock.json` and commits with the message in `bump`. Then when you push the version gets read and the image gets tagged based on that.

Reply:
> That's just a default

Ah that makes sense